students/15th/eunjinlee/main/index.html

```diff
@@ -0,0 +1,257 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://use.typekit.net/sqv7udc.css">
+  <title>Westagram Main</title>
+</head>
+<body>
+
+  <header class="header">
+    <div class="nav">
+      <p class="westragram-logo">Westagram</p>
+      <div class="nav__search-container">
+        <input type="text" class="nav__search-input" id="nav-search-input" placeholder="Search">
+        <i class="fas fa-search nav__search-icon"></i>
+        <ul class="search-list-container" id="search-list-container">    </ul>
+      </div>
+
+      <div class="nav__menu-container">
+        <img src="../images/home.png" alt="Home" class="nav__home">
```

Comment:
```suggestion
        <img alt="Home" class="nav__home" src="../images/home.png">
```
크리티컬한 부분은 아니지만 순서를 이렇게 바꿔주면 코드를 읽으면서 이 태그가 어떤 역할을 하는지 보다 빠르게 파악할 수 있을 것 같습니다

Reply:
항상 태그 작성 시마다 마구잡이식으로 작성했던 부분이었는데 조언 감사합니다